build.sbt

```diff
@@ -85,9 +85,14 @@ addCommandAlias("rebuildEnums", "; sql/flywayClean; sql/flywayMigrate; genEnums;
 addCommandAlias("schemaSpy", "sql/runMain org.schemaspy.Main -t pgsql -port 5432 -db gem -o modules/sql/target/schemaspy -u postgres -host localhost -s public")
 addCommandAlias("gemctl", "ctl/runMain gem.ctl.main")//
 
+resolvers in ThisBuild +=
+  Resolver.sonatypeRepo("public")
+
 resolvers in ThisBuild +=
   Resolver.sonatypeRepo("snapshots")
 
+updateOptions in ThisBuild := updateOptions.value.withLatestSnapshots(false)
```

Comment:
If we're going to require tagging every update, these settings may not be necessary.